src/Features/CSharp/Portable/UsePatternMatching/CSharpAsAndNullCheckCodeFixProvider.cs

```diff
@@ -45,18 +45,31 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             return SpecializedTasks.EmptyTask;
         }
 
-        private void AddEdits(
+        private static ExpressionSyntax GetCondition(SyntaxNode node)
+        {
+            switch (node.Kind())
+            {
+                case SyntaxKind.WhileStatement:
+                    return ((WhileStatementSyntax)node).Condition;
+                case SyntaxKind.IfStatement:
+                    return ((IfStatementSyntax)node).Condition;
+                default:
+                    return null;
```

Comment:
It shoudl be available:

![image](https://cloud.githubusercontent.com/assets/4564579/26800222/61515bd6-49ed-11e7-9ec1-fb09107b2016.png)